app/[user]/page.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+// These imports just for grabbing the static data - these should be removed
```

Comment:
In the future use `TODO`. It makes it easier to search the entire codebase for things you need to fix before merging.
```suggestion
// TODO: These imports just for grabbing the static data - these should be removed
```

Reply:
Didn't know about that feature THANKS